js/components/close-caption-multi-audio-menu/closeCaptionTab.js

```diff
@@ -0,0 +1,39 @@
+var React = require('react');
+var BaseTab = require('../base-components/listWithChoice');
+
+var CloseCaptionTab = React.createClass({
+  componentWillMount: function () {
+    this.header = 'Subtitles';
+    this.list = [];
+    this.updateList(this.props);
+  },
+
+  componentWillUpdate: function (nextProps, nextState) {
+    this.updateList(nextProps);
+  },
+
+  updateList: function (props) {
+    var isSelected;
+    if(props.closeCaptions && props.closeCaptions.list){
```

Comment:
Spacing. Please use spaces around the parenthesis: 
```if (//condition) {```
There are a few statements without spaces in the PR.

Also, the variable name should be `closedCaptions`